src/main/java/org/openrewrite/kotlin/tree/K.java

```diff
@@ -1036,9 +1036,8 @@ public FunctionType(UUID id, Space prefix, Markers markers, List<Annotation> lea
         }
 
         public Space getPrefix() {
-            // For backwards compatibility with older LST before there was a prefix field
             //noinspection ConstantConditions
-            return prefix == null ? returnType.getPrefix() : prefix;
+            return prefix == null ? returnType.getElement().getPrefix() : prefix;
```

Comment:
I suspect we actually want a physical `prefix` field. But I am also not 100% sure, so I suggest we ask @sambsnyd for guidance.